packages/yourloops/components/header-bars/main-header.tsx

```diff
@@ -116,8 +164,33 @@ function MainHeader({ withShrinkIcon, onClickShrinkIcon }: MainHeaderProps): JSX
             </Link>
           </Box>
 
+          {user.isUserHcp() &&
+            <StyledTabs value={selectedTab} onChange={handleTabChange} centered>
+              <StyledTab
+                data-testid="main-header-hcp-patients-tab"
+                className={tab}
+                label={t('header-tab-patients')}
+                value={HcpNavigationTab.Patients}
+                onClick={() => {
+                  handleTabClick(HcpNavigationTab.Patients)
+                }}
+              />
+              {shouldDisplayCareTeamTab &&
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🤓 nitpick:** ‏
```suggestion
              {selectedTeam.id !== PRIVATE_TEAM_ID &&
```
`shouldDisplayCareTeamTab` checks `user.isUserHcp()` but this is already done in the dom (line 137)